src/test/resources/features/ifs/MultipeDebtItems.feature

```diff
@@ -137,7 +137,10 @@ Feature: Multiple Debt Items
     And the 1st debt summary will contain
       | interestBearing | numberChargeableDays | interestDueDailyAccrual | totalAmountIntDuty |
       | true            | 0                    | 71                      | 1000000            |
-    And the 1st debt summary will not have any calculation windows
+    And the 1st debt summary will have calculation windows
+      | periodFrom | periodTo   | numberOfDays | interestRate | interestDueDailyAccrual | unpaidAmountWindow |
+      | 2022-02-02 | 2022-02-02 | 0            | 2.6          | 71                      | 1000000             |
+
```

Comment:
if interestStartDate is same as interestRequestTo , then also we dont have calc windows?

Reply:
No. We do have it then.